test/solvers/advanced/proximal_operator_test.py

```diff
@@ -464,5 +466,76 @@ def test_proximal_convconj_kl_product_space():
     assert all_almost_equal(x_verify, x_opt)
 
 
+def test_proximal_convconj_kl_version_two():
+    """Proximal factory of convex conjugate for 2nd version KL divergence."""
+
+    # Image space
+    space = odl.uniform_discr(0, 1, 10)
+
+    # Element in image space where the proximal operator is evaluated
+    x = space.element(np.arange(-5, 5))
+
+    # Data
+    g = space.element(np.arange(10, 0, -1))
+
+    # Factory function returning the proximal operator
+    lam = 2
+    prox_factory = proximal_cconj_kl_version_two(space, lam=lam, g=g)
```

Comment:
I'm gonna keep this, since it is used in the explicit computations to verify the evaluation further down.